patronload/cli.py

```diff
@@ -4,20 +4,24 @@
 
 import click
 
-from patronload.config import configure_logger, configure_sentry
+from patronload.config import configure_logger, configure_sentry, load_config_values
 
 logger = logging.getLogger(__name__)
 
 
 @click.command()
-@click.option(
-    "-v", "--verbose", is_flag=True, help="Pass to log at debug level instead of info"
-)
-def main(verbose: bool) -> None:
+def main() -> None:
     start_time = perf_counter()
+    config_values = load_config_values()
     root_logger = logging.getLogger()
-    logger.info(configure_logger(root_logger, verbose))
+    log_level = config_values["LOG_LEVEL"] or "INFO"
```

Comment:
I don't think the `or "INFO"` is needed here since it defaults to `INFO` in config.